### PR TITLE
added saver ms

### PR DESCRIPTION
**Pull request by BenSterenson**

### CODE REVIEW

File: server/data/msg_queue_data.py [hunk 1]
```diff
@@ -31,19 +37,40 @@ class _BaseMessage(ABC):
     _cls: MessageClassTypes = field(init=False)
 
     def to_doc(self) -> Dict:
-        raw_data_doc = asdict(self).copy()
+        item_copy = copy.deepcopy(self)
```
Comment (MPalarya) [inline server/data/msg_queue_data.py:40]: can we share the same logic between `baseData` and `baseMessage`?

Comment (MPalarya) [reply server/data/msg_queue_data.py:40]: and why is it different at all?

**Review by MPalarya — COMMENTED** [review]
notice:
* server/converters_micro_service/converters
* server/saver

File: server/data/msg_queue_data.py (hunk 1 above)
Comment (BenSterenson) [reply server/data/msg_queue_data.py:40]: i would like to change later baseMessage to use protobuf next, furthermore base data has different fields.

File: server/saver/run.py [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+channel.exchange_declare(exchange=msg_queue_config.get("EXCHANGE"), exchange_type=msg_queue_config.get("EXCHANGE_TYPE"))
+channel.queue_declare(queue=msg_queue_config.get("QUEUE_NAME"), durable=True)
+channel.queue_bind(exchange=msg_queue_config.get("EXCHANGE"), queue=msg_queue_config.get("QUEUE_NAME"))
+
+print(" [*] Waiting for messages. To exit press CTRL+C")
+
+
+def should_handle_message(message: BaseMessage) -> bool:
+    return message.recipient == MessageRecipientTypes.SAVER
```
Comment (MPalarya) [inline server/saver/run.py:28]: this is the only different line compared to converter's run.py
can we do something about it? extract it to a class in a common folder such that we can only override this method?

File: server/data/msg_queue_data.py (hunk 1 above)
Comment (MPalarya) [reply server/data/msg_queue_data.py:40]: we can create a common parent for them

File: server/saver/run.py (hunk 2 above)
Comment (BenSterenson) [reply server/saver/run.py:28]: forgot also to add the cast part, I think its an overkill lets talk about it.

File: dockers/converter_settings.yaml [hunk 3]
```diff
@@ -1,17 +1,17 @@
 REDIS:
-  HOST: fin-redis
-  PORT: 6379
+  HOST: "fin-redis"
+  PORT: "6379"
```
Comment (BenSterenson) [inline dockers/converter_settings.yaml:3]: i think i saw warnings when we used port as strings i think the client expects int

File: dockers/converter_settings.yaml [hunk 4]
```diff
@@ -1,17 +1,17 @@
...
+  URI: "mongodb://fin-mongo:27017"
+  FINANCE_DB: "finance"
+  USER_COLLECTION: "users"
+  RAW_DATA_COLLECTION: "raw_data"
+  NORMELIZED_DATA_COLLECTION: "normalized_data"
 
 MESSAGE_QUEUE:
   HOST: "fin-rabbitmq"
-  PORT: 5672
+  PORT: "5672"
```
Comment (BenSterenson) [inline dockers/converter_settings.yaml:14]: same here

**Review by MPalarya — APPROVED** [review]

**Pull request merged**